README.md

```diff
@@ -2,32 +2,19 @@
 
 ThreatExchange is a set of RESTful APIs on the Facebook Platform for querying, publishing, and sharing security threat information. It's a lightweight way for exchanging details on malware, phishing pages, and other threats with either specific members of the community or the ThreatExchange community at large.
```

Comment:
Do we need to add a "This repository contains a collection of sub-projects that are affiliated with ThreatExchange, or support data formats shared on ThreatExchange" type statement somewhere here, to into into subprojects?

Reply:
Added H2 header right below. I think makes it clear